docs/cubeviz/plugins.rst

```diff
@@ -41,6 +41,13 @@ the wavelength entry will update to the exact value of that slice).
 
 The spectrum viewer also contains a tool to allow clicking and 
 dragging in the spectrum plot to choose the currently selected slice.
+If another toolbar icon is selected, and then deselected after use,
```

Comment:
Maybe it is best if @kecnry can look at this since he implemented this feature.

Reply:
right now this is technically true, but will change slightly if/when #1115 is merged.   Maybe it would be simpler to just skip this part about deselecting, etc, and say "When the slice to is active, clicking anywhere on the spectrum viewer will select the nearest slice across all viewers, even if the indicator is off-screen"?  The underlying logic for "defaulting" tools is somewhat complex to explain in words, but I think is intuitive and quickly learned just by using it 🤞

Reply:
Sounds good, I updated the text in the latest commit. Thanks!